### PR TITLE
Uninstall 42090

### PR DESCRIPTION
**Pull request by dian-xh**
changes
- fixes 42090
- switched csv to 3.2.4

tests
- local smoke test

### CODE REVIEW

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+          }
+        }
+      ]
+    capabilities: "Basic Install"
+    "operatorframework.io/suggested-namespace": ibm-common-services
+    categories: "Logging & Tracing"
+    certified: "false"
+    support: "IBM"
+    repository: https://github.com/IBM/ibm-elastic-stack-operator
+    image: quay.io/opencloudio/ibm-elastic-stack-operator:3.2.3
```
Comment (bluzarraga) [inline deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml:194]: does this need to be 3.2.4?

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+  maturity: alpha
+  provider:
+    name: IBM
+  links:
+  - name: Elastic
+    url: https://www.elastic.co/
+  - name: Elastic Stack Operator
+    url: https://github.com/IBM/ibm-elastic-stack-operator
+  replaces: ibm-elastic-stack-operator.v3.2.2
+  version: 3.2.3
```
Comment (bluzarraga) [inline deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml:525]: 3.2.4?

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: ibm-elastic-stack-operator
+                # 3.2.3
+                # image: quay.io/opencloudio/ibm-elastic-stack-operator@sha256:9b21af3f4e3cbc4342921297197f331dd3dba839db914af1697ccf5c5e70dd43
+                image: quay.io/opencloudio/ibm-elastic-stack-operator:3.2.3
```
Comment (bluzarraga) [inline deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml:486]: 3.2.4?

Comment (dian-xh) [reply deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml]: updated. thanks!

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml (hunk 2 above)
Comment (dian-xh) [reply deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml]: updated. thanks!

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml (hunk 1 above)
Comment (dian-xh) [reply deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml]: updated. thanks!

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+metadata:
+  name: ibm-elastic-stack-operator.v3.2.3
+  labels:
+    name: ibm-elastic-stack-operator
+    app.kubernetes.io/instance: common-logging
+    app.kubernetes.io/managed-by: elasticstacks.elasticstack.ibm.com
+    app.kubernetes.io/name: elasticstacks.elasticstack.ibm.com
+  namespace: placeholder
+  annotations:
+    olm.skipRange: '>=3.0.5 <3.2.3'
```
Comment (bluzarraga) [inline deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml]: 3.2.4

File: deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,525 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-elastic-stack-operator.v3.2.3
```
Comment (bluzarraga) [inline deploy/olm-catalog/ibm-elastic-stack-operator/3.2.4/ibm-elastic-stack-operator.v3.2.4.clusterserviceversion.yaml]: 3.2.4

**Pull request merged**